### PR TITLE
chore: update demo project

### PR DESCRIPTION
**Pull request by Foxandxss**
Having a different repo for the docs was a dead end. Webpack will get crazy when trying to import stuff from a different project.

I updated what we got using a simplified version of my starter. No lint, no test. Will add those when the time is right, but we need something to show like for yesterday.

As soon as I merge this, I will make 2-3 demos so @icfantv can play with the CSS

### CODE REVIEW

**Comment by icfantv** [comment]
That really stinks.  So we have to have everything under core?  What happens when we break out components?  Is that going to cause the same issue?

**Comment by Foxandxss** [comment]
We will cross that bridge when that happens.

Perhaps when we push this repo to npm, we can install it into the demo project and work with it, but as today, I don't find a clean way.

File: webpack.demo.js [hunk 1]
```diff
@@ -1,39 +1,269 @@
...
+  }
+
+  /**
+   * PostCSS
+   * Reference: https://github.com/postcss/autoprefixer-core
+   * Add vendor prefixes to your css
+   */
+  config.postcss = [
+    autoprefixer({
+      browsers: ['last 2 version']
```
Comment (wesleycho) [inline webpack.demo.js]: Isn't this config default with autoprefixer?

File: webpack.demo.js [hunk 2]
```diff
@@ -1,39 +1,269 @@
...
+    historyApiFallback: true,
+    stats: 'minimal' // none (or false), errors-only, minimal, normal (or true) and verbose
+  };
+
+  return config;
+}();
+
+// Helper functions
+function root(args) {
+  args = Array.prototype.slice.call(arguments, 0);
```
Comment (wesleycho) [inline webpack.demo.js]: No need for this, you can do `function root(...args) { ... }` - remember, webpack runs in Node, you have ES6!

File: webpack.demo.js [hunk 3]
```diff
@@ -1,39 +1,269 @@
...
+  return path.join.apply(path, [__dirname].concat(args));
+}
+
+function rootNode(args) {
+  args = Array.prototype.slice.call(arguments, 0);
```
Comment (wesleycho) [inline webpack.demo.js]: Ditto

File: webpack.demo.js [hunk 4]
```diff
@@ -1,39 +1,269 @@
...
+    stats: 'minimal' // none (or false), errors-only, minimal, normal (or true) and verbose
+  };
+
+  return config;
+}();
+
+// Helper functions
+function root(args) {
+  args = Array.prototype.slice.call(arguments, 0);
+  return path.join.apply(path, [__dirname].concat(args));
```
Comment (wesleycho) [inline webpack.demo.js]: You can do `[__dirname].push(...args)` here - this is the spread operator

File: webpack.demo.js [hunk 5]
```diff
@@ -1,39 +1,269 @@
...
+}
+
+function rootNode(args) {
+  args = Array.prototype.slice.call(arguments, 0);
+  return root.apply(path, ['node_modules'].concat(args));
```
Comment (wesleycho) [inline webpack.demo.js]: `['node_modules'].push(...args)`

**Comment by icfantv** [comment]
@Foxandxss just lemme know when you're ready and I'll get started

**Comment by wesleycho** [comment]
After discussion, this LGTM

**Comment by icfantv** [comment]
I'm ok with this.

**Pull request closed without merging**